[Alerts][Doc] Added README documentation for alerts plugin status and framework health checks configuration options. (#92761)
elastic/kibana: x-pack/plugins/alerts/README.md

    is unauthorized for user [user-name-here]
```

## Plugin status

> Alerts plugin's status is customized by including information about checking framework decryption failures:

Reviewer: ```suggestion
The plugin status of an alert is customized by including information about checking failures for the framework decryption:
```

```
core.status.set(
        combineLatest([
          core.status.derivedStatus$,
          getHealthStatusStream(startPlugins.taskManager),
        ]).pipe(
          map(([derivedStatus, healthStatus]) => {
            if (healthStatus.level > derivedStatus.level) {
              return healthStatus as ServiceStatus;
            } else {
              return derivedStatus;
            }
          })
        )
      );
```

> To check framework decryption failures we use task `alerting_health_check`, which runs with default interval 60 mins. To change the dafault schedule use kibana.yml configuration option `xpack.alerts.healthCheck.interval: 60m`.

Reviewer: ```suggestion
To check for framework decryption failures, we use the task `alerting_health_check`, which runs every 60 minutes by default. To change the default schedule, use the kibana.yml configuration option `xpack.alerts.healthCheck.interval`.
```
